### ISSUE TITLE
Gamelist 

### ISSUE THREAD

**Issue by Titiclash** [TapiocaFox/Daijishou#737]
I added games to Daijisho, Mame complete collection, as there are repeated bios  ZZZ games, there are thousands of games, I didn't find an option to delete games in bulk from the list, I can only delete them one by one, it would be a good point of improvement to this project that is incredible

**Comment by 7ank0v1c** [comment]
I can picture like a checkbox screen, with some functions to select and maybe something else like scrape? Might be a worthy addition

**Comment by Titiclash** [comment]
> Posso imaginar como uma tela de caixa de seleção, com algumas funções para selecionar e talvez algo mais como raspar? Pode ser uma adição valiosa

exactly, it would be an incredible option, I have to make a bat file to remove the ZZZ file games that are not found by emulationstation
